composer.json

```diff
@@ -25,7 +25,7 @@
     "require-dev": {
         "guzzlehttp/guzzle": "^7",
         "guzzlehttp/psr7": "^2",
-        "openeuropa/code-review": "^2.0.0-alpha6",
+        "openeuropa/code-review": "^2.0",
```

Comment:
if we upgrade the code review component then we need to use the new one for libraries https://github.com/openeuropa/code-review-library